server/controllers/chartsController.js

```diff
@@ -104,6 +107,12 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
           targetTotal: newTargetValue,
         }
       }
+      studyTotals[constants.TOTALS_STUDY].targetTotal =
+        !!newTargetValue && !!studyTotals[constants.TOTALS_STUDY].targetTotal
+          ? studyTotals[constants.TOTALS_STUDY].targetTotal + newTargetValue
+          : !!newTargetValue && !studyTotals[constants.TOTALS_STUDY].targetTotal
+          ? newTargetValue
+          : studyTotals[constants.TOTALS_STUDY].targetTotal
```

Comment:
can we make this so it isn't a nested ternary? Those can be super hard to reason through 😄

Reply:
+1 🙃

Reply:
https://github.com/AMP-SCZ/dpdash/pull/147/commits/04f13bd1ffa2be4dabbec2ecb9bb841fb2abafa3